src/cms/cms.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react'
+import CMS from "netlify-cms-app"
+import LargeFactCard from "../components/large-fact-card"
+import { makeStyles } from '@material-ui/core/styles';
+import injectStyle from './injectStyle';
+
+const factPageStyles = makeStyles({
+  card: {
+    maxWidth: 520,
+    marginTop: 20,
+    margin: 'auto',
+  },
+});
+
+const FactPagePreview = ({ entry, widgetFor, getAsset }) => {
```

Comment:
If we end up adding additional types of pages we will want to split this into its own file.